documentation/docs/upgrade-v0.8.5.md

```diff
@@ -0,0 +1,33 @@
+---
+id: upgrace-v0.8.5
+sidebar_position: 10
+sidebar_label: Upgrade to v0.8.5
+slug: /upgrade-v0.8.5
+---
+
+# How to upgrade your node to v0.8.5
+
+Please perform this process sometime before 14:30 UTC on Friday, 2021-nov-26.
+
+## Ordinary (non-validator) nodes: Docker
+
+TL;DR
+```
+docker stop axelar-core
```

Comment:
You should add the instructions for anyone using the binaries too. The setup doc page has the command to kill the corresponding processes.